### ISSUE TITLE
[1.38.0-beta.1]  Bold and Italics buttons in dismiss issue comments missing aria-pressed

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1469]
The bold and italics buttons do not communicate their pressed state to screen reader users.
<img width="719" height="433" alt="Image" src="https://github.com/user-attachments/assets/405c78f1-3dd8-4168-b941-ef82f4b8e122" />

<img width="696" height="439" alt="Image" src="https://github.com/user-attachments/assets/8d06a548-10e0-4e8c-9ee2-f03fb9a9d02b" />

```
 <button type="button" class="components-button is-small has-icon" aria-label="Bold (Ctrl+B)"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" size="24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M14.7 11.3c1-.6 1.5-1.6 1.5-3 0-2.3-1.3-3.4-4-3.4H7v14h5.8c1.4 0 2.5-.3 3.3-1 .8-.7 1.2-1.7 1.2-2.9.1-1.9-.8-3.1-2.6-3.7zm-5.1-4h2.3c.6 0 1.1.1 1.4.4.3.3.5.7.5 1.2s-.2 1-.5 1.2c-.3.3-.8.4-1.4.4H9.6V7.3zm4.6 9c-.4.3-1 .4-1.7.4H9.6v-3.9h2.9c.7 0 1.3.2 1.7.5.4.3.6.8.6 1.5s-.2 1.2-.6 1.5z"></path></svg></button>
      <button type="button" class="components-button is-small has-icon" aria-label="Italic (Ctrl+I)"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" size="24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M12.5 5L10 19h1.9l2.5-14z"></path></svg></button>
```

Add an `aria-pressed` attribute and toggle the value as needed.
